examples/settings.py

```diff
@@ -8,16 +8,22 @@
 
 use_interactive_JSON_viewer = False
 
-# Account settings. Need a one-time adjustment for examples to work.
+# Account settings. Need a one-time adjustment for examples to work. These should be set in settings.json
 
     # ACCOUNT_ID: Account ID. See get_authentication_params.ipynb example for more information.
     # AUTH_TOKEN: Account authentication token. See get_authentication_params.ipynb for more information.
     # MATERIALS_PROJECT_API_KEY: Materials project API key. See https://materialsproject.org/open for more information.
 
-ACCOUNT_ID = "ACCOUNT_ID"
-AUTH_TOKEN = "AUTH_TOKEN"
-MATERIALS_PROJECT_API_KEY = "MATERIALS_PROJECT_API_KEY"
-ORGANIZATION_ID = "ORGANIZATION_ID"
+# Load variables from the settings.json file
+import json, os
+relative_path_to_settings_json_file = os.path.join(os.path.dirname(__file__), 'settings.json')
```

Comment:
Seems like `absolute path to me, not relative`

Reply:
You are right, sorry. This is absolute path. I found that the path during runtime is at the file that is importing settings.py, so this path should be `absolute_path_to_settings_json_file`